### ISSUE TITLE
Style guide review

### ISSUE THREAD

**Issue by jenniferthibault** [18F/18f.gsa.gov#2383]
👋 🎉 @gemfarmer @elainekamlley  I was finally able to take a peek at the latest! I'm looking at mostly small things that would detract from the review experience. Once we're able to take care of these I think it would be in a good place to bring to other design folks (Amber, Austin + TBD) for a round of feedback.

Let me know if you want to pair up or have questions on any of this!

## Side nav
- [x] **Navigation through the side nav should allow you to get to the landing page for that section**
- Expected: When clicking any of the expandable sections (Typography, Images, Components..) When I clicked the parent category, I expected to be taken to the landing section of that page, and for the accordion for the section to open. Example: [the USWDS](https://standards.usa.gov/components/typography/)
- Actual: The accordion opens, but you stay on the page you were on before. This makes it easy to miss the opening content for the section.

- [x] **Active states in the side nav should be accurate**
- Expected: When navigating to a sub-section element, the side nav will change to show me which section I'm in.
- Actual: In some sections, the active state will be on a different section of the side nav than the page you're on
  - <img width="1423" alt="screen shot 2017-04-03 at 9 17 07 am" src="https://cloud.githubusercontent.com/assets/11636908/24611444/1f5036c2-1850-11e7-8026-93b41639712b.png">
  - <img width="1416" alt="screen shot 2017-04-03 at 9 28 41 am" src="https://cloud.githubusercontent.com/assets/11636908/24611446/1f5361c6-1850-11e7-9112-2c40e1b551d4.png">

- [x] **Current section indicator should reach top and bottom lines of the container**
- Expected: the vertical blue bar should fit exactly between the top and bottom lines
- Actual: It's a smidge too short, and misses the top bar by a hair. This catches the viewers' eye and is a slight distraction.
  - <img width="246" alt="screen shot 2017-04-03 at 9 46 24 am" src="https://cloud.githubusercontent.com/assets/11636908/24612183/7ff78b5e-1852-11e7-9bf6-21dc5e4ebf5e.png">

- [x] **Menu item borders should be equal weight**
- Actual: it seems like the borders between expandable items are thicker than the others
<img width="251" alt="screen shot 2017-04-03 at 9 58 37 am" src="https://cloud.githubusercontent.com/assets/11636908/24612851/6f3d10de-1854-11e7-8d83-0555bbce2b10.png">

## Typesetting
- [x] **Remove huge space above h3 class**
- Expected: h3 elements have spacing set proportional to their hierarchy on the page
- Actual: h3 elements look like they have a `padding-top: 13rem` on them, coming from the `.styleguide-content > h3` class. This space breaks up the layout in unexpected ways when attached to an h class. It's especially confusing when seeing in the typesetting section, because it makes it seem like all h3s are intended to have this space, which seems to only be intended for the style guide. Might be good to chat about what that space is there for and how to build it in without attaching it to the h class?
  - <img width="358" alt="screen shot 2017-04-03 at 9 33 08 am" src="https://cloud.githubusercontent.com/assets/11636908/24611651/d64a450c-1850-11e7-899f-4dc433d96449.png">

## Image examples
- [x] **Failed images should demonstrate their states**
- Actual: The images render the same as the source reference, but I think they aren't supposed to?
  - <img width="453" alt="screen shot 2017-04-03 at 9 49 32 am" src="https://cloud.githubusercontent.com/assets/11636908/24612332/035be2f6-1853-11e7-8db4-e249a2fd0c6b.png">

**Comment by gemfarmer** [comment]
@jenniferthibault as always, this is amazing! 🤗 

These changes all seem very reasonable. When do you think we should plan on scheduling a design review?

For my own purposes, a quick list of changes:
- [ ] Sidnav
  - [ ] landing, "overview" link
  - [ ] active state
  - [ ] ugly gap between blue active indicator
  - [ ] unequal border weight on accordion vs non-accordion
- [ ] h3 padding issue
- [ ] Failed image examples

@elainekamlley

**Comment by jenniferthibault** [comment]
@gemfarmer I'd say as soon as these little buggies are squashed and we have a branch that can show that state, I think it's ready to show! (What I'm trying to say is that merging to master & having it on production isn't a blocker, but having it in one place is.) I think it would be good to get these taken care of before the review so that we can get a "clean" view for feedback on things that aren't just accidents.

One thing that will help set us up for feedback is knowing what we'd like feedback on: setting parameters for folks to focus on. Do you have a sense of what those are? Example:
- Feedback that focuses on style & presentation only?
- Feedback that focuses on content?
- Feedback that focuses on structure?
- Feedback on everything & anything?

**Comment by gemfarmer** [comment]
@jenniferthibault I made some changes in PR #2388. Check 'em out!

#### tldr; Basically everything and anything, but with an emphasis on who the user is, and what kind of structure the guide should take to help them on their way.

As for feedback, those are all good questions. What I'm most concerned with is having a clearly defined audience and making sure that the guide caters to those users. Right now I see the two major users as:
1. People who are working on 18f.gsa.gov and need a reference for how to work within the site
2. People who are on other projects who want to see effective ways for combining 18F brand and USWDS. 

If there are additional users who we want to target, I think that figuring who they are soon willl help ensure that we are doing work that benefits those people.

Other than that, I would also appreciate feedback on how to structure the documentation so that it is informative and predictable. Right now some sections are great and others feel like they miss the mark, but I'm not entirely sure why or how.

**Comment by gemfarmer** [comment]
[Slack conversation about style guide review](https://gsa-tts.slack.com/archives/C0MKZ8E9H/p1491926816194686)

**Summary**. We are going to break up the review process into a few steps:
1. **v5** – User interview outreach team
2. **v6* *– User interview folks from guides and subdomains
3. **v7* *– Design review

**Comment by jenniferthibault** [comment]
Awesome! Since the style & design feedback points above were integrated by https://github.com/18F/18f.gsa.gov/issues/2388 , you do want to close this issue out and start a new one for research planning?

**Comment by gemfarmer** [comment]
Good idea. Closing with #2391
